pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.socialexplorer</groupId>
     <artifactId>fastdbf4j</artifactId>
-    <version>1.0.7</version>
+    <version>1.0.8</version>
```

Comment:
need also to change the other files, checksums, etc. check previous commit, i can help you tomorrow with it. we basically use github as a maven repository, we hacked that.